android/java/org/chromium/chrome/browser/crypto_wallet/util/AddressUtils.java

```diff
@@ -23,6 +23,7 @@ public static String sanitizeEthAddress(String ethAddress) {
      */
     public static String getTruncatedAddress(String address) {
         int prefixLength = address.startsWith("0x") ? 6 : 4;
-        return (address.substring(0, prefixLength) + "…" + address.substring(address.length() - 4));
+        return (address.substring(0, prefixLength) + "***"
```

Comment:
non security blocking nit: `*` charter makes sense to me here. Does utilizing a larger character have any side effects on other places where this function gets used such that addresses wrap in unwanted ways? Not sure what rules we have about it, but figured you would know the style guides better here so wanted to call it out.

Reply:
E.g.

`1234....1234` is single line is generally expected

```
1234****12
34
```

basically just wanted to double check if this might trigger some test wrap like above.

Reply:
We were showing full address in multiple lines but now we started to show the partial address like desktop which is even shorter so it should fit in one line.